docs/source/performance.md

```diff
@@ -0,0 +1,75 @@
+# Performance tuning
+
+1.6 Scylla Operator adds a new experimental feature allowing users to optimize Kubernetes nodes
+
+## Node tuning
+
+From 1.6 a new CRD called NodeConfig is available which allows users to target Nodes which should be tuned.  
+Nodes matching the provided placement conditions will be subject for tuning.
+
+Below example NodeConfig tunes nodes having `scylla.scylladb.com/node-type=scylla` label:
+```
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: NodeConfig
+metadata:
+ name: cluster
+spec:
+ placement:
+   nodeSelector:
+     scylla.scylladb.com/node-type: scylla
+```
+For more details about new CRD use:
+```
+kubectl explain nodeconfigs.scylla.scylladb.com/v1alpha1
+```
+
+Tuning consists of two separate optimizations: common tuning of node, and tuning based on Scylla Pods and their resource assignment.
+First optimization is executed when tuning Pod lands on Node. Second one is executed when Scylla Pod lands on the same Node.  
+Tuning Pods are created in special namespace called `scylla-operator-node-tuning`
+
+Only Scylla Pods having Guaranteed Quality of Service class are tuned, so make sure your ScyllaCluster meets all necessary conditions.
```

Comment:
The tuning is applied only to pods with *guaranted* QoS. Please double check you ScyllaCluster resource specifications.

Reply:
done